nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-nar-utils/src/main/java/org/apache/nifi/nar/NarThreadContextClassLoader.java

```diff
@@ -192,7 +192,7 @@ public static NarThreadContextClassLoader getInstance() {
     public static <T> T createInstance(final ExtensionManager extensionManager, final String implementationClassName, final Class<T> typeDefinition, final NiFiProperties nifiProperties)
             throws InstantiationException, IllegalAccessException, ClassNotFoundException {
         final ClassLoader originalClassLoader = Thread.currentThread().getContextClassLoader();
-        Thread.currentThread().setContextClassLoader(NarThreadContextClassLoader.getInstance());
+        //Thread.currentThread().setContextClassLoader(NarThreadContextClassLoader.getInstance());
```

Comment:
Is this on purpose?  Or a left over from debugging?

Reply:
It's on purpose, but I'll remove the dead code. We want the NAR's classloader to be used to load things, not the singleton

Reply:
OK, that makes sense, just wasn't sure with the commented out code ;)